_posts/2020-10-27-scalajs-for-scala-3.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: "Implementing Scala.js Support for Scala 3"
+---
+
+Today, the JS platform has become an integral part of the language.
```

Comment:
I initially read the "Today," as meaning that something literally happened today. I think "These days," or perhaps "In recent years" ("In der letzte Zeit"?) would be a bit better.